src/features/machine/polarMachine.js

```diff
@@ -0,0 +1,210 @@
+import { angle, onSegment } from '../../common/Geometry'
+import Machine from './machine'
+import Victor from 'victor'
+
+export const traceCircle = (start, end, size) => {
```

Comment:
I had to change the function signature, since I didn't have an object with a .angle in my code. I will push a fix.